app/services/delivery_partners/api/query.rb

```diff
@@ -0,0 +1,19 @@
+module DeliveryPartners::API
```

Comment:
I wonder if it might be better to move this subclass and others to `api/#model_name/query` as that would sit nicely with the existing structure?